cmd/ocsp.go

```diff
@@ -0,0 +1,165 @@
+/*
+The MIT License (MIT)
```

Comment:
We can make a package out of this.. since it doesn't belong directly in `cmd` .. you can put it under pkg/ocsp